app/team.go

```diff
@@ -1906,6 +1906,24 @@ func (a *App) InvalidateAllEmailInvites() *model.AppError {
 	if err := a.Srv().Store.Token().RemoveAllTokensByType(TokenTypeGuestInvitation); err != nil {
 		return model.NewAppError("InvalidateAllEmailInvites", "api.team.invalidate_all_email_invites.app_error", nil, err.Error(), http.StatusBadRequest)
 	}
+	if err := a.InvalidateAllResendInviteEmailJobs(); err != nil {
+		return model.NewAppError("InvalidateAllEmailInvites", "api.team.invalidate_all_email_invites.app_error", nil, err.Error(), http.StatusBadRequest)
+	}
```

Comment:
@AGMETEOR it seems this change is disabling any pending reminders correct? However, based on the ticket the invitation link still worked in the reminder email? Shouldn't the line above had invalidated the token of the invitation link of the reminder as well?

Reply:
Not really. In fact what we’re doing here is to stop any reminder jobs from running again. The only reason, the token from reminder is valid is because everytime that job runs, it creates a new token.

Reply:
Oh ok, that makes sense! Thanks for clarifying 👍